docker/Dockerfile

```diff
@@ -42,7 +43,7 @@ RUN conda install -y python=${python_version} && \
     pip install \
       sklearn_pandas \
       tensorflow-gpu && \
-    pip install https://cntk.ai/PythonWheel/GPU/cntk-2.1-cp36-cp36m-linux_x86_64.whl && \
```

Comment:
@tedyu, Thank you for the PR. CNTK is pip-installable now. Please replace the url with just `cntk`.

Reply:
Sorry.
Somehow I missed the above comment :-)